data/scripts/sol-oic-gen.py

```diff
@@ -430,18 +430,18 @@ def generate_object_from_repr_vec_fn_common_c(name, props):
     const struct sol_oic_map_reader *repr_vec, uint32_t decode_mask)
 {
     struct sol_oic_repr_field field;
-    enum sol_oic_map_loop_status end_status;
```

Comment:
But reason outnumbers status throughout soletta for these loop conditions. Also it differentiates from other enums and such that really carry status for handles. What do we do here? Change all the other reason entries?